### ISSUE TITLE
Middlescroll and downscroll not compatible

### ISSUE THREAD

**Issue by DiosDonutSlave** [KadeDev/Funkin#2591]
Using downscroll, and middlescroll doesn't work. It just looks like downscroll with 2 lanes instead of one.
![16351328066841435658583247384925](https://user-images.githubusercontent.com/80781504/138630835-d9832096-7077-4139-a694-b402ea1ebe3a.jpg)
![16351328243894663792239465922305](https://user-images.githubusercontent.com/80781504/138630859-59354c0a-e211-4853-a338-e7c6c5f5d1b7.jpg)
![16351328379064715843317118837136](https://user-images.githubusercontent.com/80781504/138630887-fcdc2a7d-97e7-4ff1-b71a-50bf79418a39.jpg)
As you see I have a custom noteskin enabled, but that shouldn't stop the middle scroll from working right?

**Comment by RagdollClash** [comment]
Tutorial can't use middle scroll as it's a modchart.

**Comment by DiosDonutSlave** [comment]
Then the friggin engine should at least support middle scroll for
modcharts??? Thats kinda broken tbh...

**Comment by ItzFireable** [comment]
> Then the friggin engine should at least support middle scroll for modcharts??? Thats kinda broken tbh...

nah. that would genuinely break so many pre existing modcharts because 1. it uses opponent side notes as well and 2. everything would be offset

that's not broken. it's intentional.
